selenium/firefox/apt/Dockerfile.tmpl

```diff
@@ -13,6 +13,7 @@ RUN  \
         apt-get update && \
         add-apt-repository -y $APT_REPO && \
         apt-get update && \
+        apt-get -y install iproute2 && \
```

Comment:
You may want to install this on the same line as Firefox, the same way it's done for Chrome's Dockerfile.
Also, you could also add iproute2 to selenium/firefox/local/Dockerfile.tmpl for consistency.